pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub, yarn2nix, makeDesktopItem, electron, riot-web }:
+
+let
+  executableName = "riot-desktop";
+  version = "1.0.4";
+  riot-web-src = fetchFromGitHub {
+    owner = "vector-im";
+    repo = "riot-web";
+    rev = "v${version}";
+    sha256 = "152mi81miams5a7l9rd12bnf6wkd1r0lyicgr35r5fq0p6z7a4dk";
+  };
+
+in yarn2nix.mkYarnPackage rec {
+  name = "riot-desktop";
```

```suggestion
  pname = "riot-desktop";
```
I'm not familiar with `mkYarnPackage` or if it accepts `pname`.
Currently the derivation name won't have a `version`.

---

The `pname` attribute isn't supported by `mkYarnPackage`.
I've manually added a version suffix to the name attribute.